### PR TITLE
Added settings quorum tests

### LINKED ISSUE #100: Threshold testing
- [x] should show members of different roles
- [x] should disable input and submit button for non authorised people
- [x] should be able to update policy by fixed vote count
- [x] should allow only valid input for threshold
- [x] should be able to update policy by percentage

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Resolves #100 
[video.webm](https://github.com/user-attachments/assets/0f7e97a9-d0b8-4d34-bbdb-d21728dd952d)
[video.webm](https://github.com/user-attachments/assets/922960e2-892c-4f1a-ac57-3421d61e24b8)
[video.webm](https://github.com/user-attachments/assets/c360207d-5f06-4238-b31d-b0576fd46544)
[video.webm](https://github.com/user-attachments/assets/f2117d33-faa7-4e29-a236-a19f8c5c5713)
[video.webm](https://github.com/user-attachments/assets/0ae319ec-1a05-4cb2-be20-07c2868304eb)

### CODE REVIEW

**Comment by race-of-sloths** [comment]
@Megha-Dev-19 Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!


[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://race-of-sloths.com/images/bot-banner-dark.svg">
    <source media="(prefers-color-scheme: light)" srcset="https://race-of-sloths.com/images/bot-banner-light.svg">
    Shows inviting banner with latest news.
</picture>](https://race-of-sloths.com/latest-news)

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/Megha-Dev-19?type=bot&pr=NEAR-DevHub/neardevhub-treasury-dashboard/101&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/Megha-Dev-19?type=bot&pr=NEAR-DevHub/neardevhub-treasury-dashboard/101&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/Megha-Dev-19)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command. Alternatively, autoscoring [1,2] will be applied for this pull request

</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/Megha-Dev-19)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Comment by petersalomonsen** [comment]
@race-of-sloths score 5

**Review by petersalomonsen — APPROVED** [review]

**Pull request merged**
